docker/Dockerfile.ubuntu18.04

```diff
@@ -75,29 +80,45 @@ RUN apt-get update && \
       iproute2 \
       ethtool \
       iptables \
-      perl \
+      netbase \
       libgssapi-krb5-2 \
-      libldap-2.4-2 \
+# Ubuntu 18/20 --> libldap-2.4-2
+# Ubuntu 22 --> libldap-2.5-0
+      libldap-2.?-? \
       libgoogle-glog0v5 \
-      libdouble-conversion1 \
-      libconfig++9v5 && \
-    rm -rf /var/lib/apt/lists/*
+# Ubuntu 18 --> libdouble-conversion1
+# Ubuntu 20/22 --> libdouble-conversion3
```

Comment:
@rdefosse you can remove the comments for ubuntu 22 from all over the dockerfile, since we don't support here